.github/CODEOWNERS

```diff
@@ -1 +0,0 @@
-*       @ahamidi
```

Comment:
I'd add a comment here that describes what this file does, maybe with a link to github docs about codeowners.

Reply:
Yup, that makes sense. :+1: